### DISCUSSION TITLE
Host Name on Dashboard

### DISCUSSION THREAD

**Discussion by feh123** [RaspAP/raspap-webgui#1044] [General]
Does anyone see the host name on Dashboard? With 2.7.5 and 2.7.6 I only see an asterisk. Is there somewhere in the UI I can add the name? Thanks.

**Comment by billz** [comment]
Hostname is displayed on the System tab. It's never been displayed on the dashboard, nor can it be set via the UI.
Lots of tutorials online on how to do the latter.

<img width="394" alt="Untitled" src="https://user-images.githubusercontent.com/229399/140415284-5a39fca8-ad20-461b-b908-a98f3ae78f9e.png">
